packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -67,25 +67,31 @@ spec:
         name: istio-ingressgateway
         type: {{ .Values.ingressGateways.type }}
     global:
+      hub: {{ .Values.global.hub | default "docker.io/istio" }}
+      tag: {{ .Values.tag }}
       istioNamespace: {{ template "istio.namespace" . }}
       proxy:
-        image: {{ .Values.global.proxy.image }}
+        image: {{ template "system_default_registry" . }}{{ .Values.global.proxy.repository }}:{{ .Values.tag }}
       proxy_init:
-        image: {{ .Values.global.proxy_init.image}}
+        image: {{ template "system_default_registry" . }}{{ .Values.global.proxy_init.repository}}:{{ .Values.tag }}
     istiocoredns:
-      coreDNSImage: {{ .Values.istiocoredns.image.repository }}
-      coreDNSPluginImage: {{ .Values.istiocoredns.pluginImage }}
+      coreDNSImage: {{ template "system_default_registry" . }}{{ .Values.istiocoredns.image.repository }}
```

Comment:
needs tag

Reply:
coredns is set like this in the manifest: 
```
image: {{ .Values.istiocoredns.coreDNSImage }}:{{ .Values.istiocoredns.coreDNSTag }}
```
since it doesn't check for the slash like pilot does, for example, I can't provide `<hub>/<registy>:<tag>`  as the value for the image. The tag is set separately on line 80 `coreDNSTag: {{ .Values.istiocoredns.image.tag }}`
```
{{- if contains "/" .Values.pilot.image }}
          image: "{{ .Values.pilot.image }}"
{{- else }}
          image: "{{ .Values.pilot.hub | default .Values.global.hub }}/{{ .Values.pilot.image | default "pilot" }}:{{ .Values.pilot.tag | default .Values.global.tag }}"
{{- end }}
```

Reply:
Ahh, gotcha, as long as it’s specified in the values it should be fine